netlify-webhook-deploy.php

```diff
@@ -348,20 +394,44 @@ public function create_plugin_settings_page() {
 
             add_menu_page( $page_title, $menu_title, $capability, $slug, $callback, $icon, $position );
         }
+
+        if ( current_user_can( $adjust_settings ) ) {
+            $sub_page_title = 'Schedule Builds';
+            $sub_menu_title = 'Schedule Builds';
+            $sub_capability = 'manage_options';
```

Comment:
This must be changed to:
```suggestion
            $sub_capability = $adjust_settings;
```
I overlooked this when I did my earlier review. 

Also, there were some changes made in the upstream in PR #12 that should be merge into the current branch (it fixes a similar issues for the two existing if-statements).

Reply:
Good call on that upstream merge. Just sorted.